### DISCUSSION TITLE
Extending front-end functionality

### DISCUSSION THREAD

**Discussion by jeannot-prvlimburg** [microsoft/sample-app-aoai-chatGPT#857] [Q&A]
Hi all, I'm currently exploring the possibility to extend the front-end with more functionalities. I would like to add the ability to select the search index and adjust the temperature. However, I have limited experience in front-end development and I am having trouble understanding the [index-3777dda7.js](https://github.com/microsoft/sample-app-aoai-chatGPT/blob/main/static/assets/index-3777dda7.js) file, which seems to control the content of the web app. This file appears to lack structure, making it challenging to modify. I believe I may be overlooking something due to my inexperience. Any assistance or guidance would be greatly appreciated.

**Comment by stbere** [comment]
Check out this repo: https://github.com/stbere/azure-search-openai-demo

Once this webapp gets deployed, it has the developer settings tab on the webpage, so the config/setup to do that is somewhere in that codebase.
